test/distributed/_composable/fsdp/test_fully_shard_training.py

```diff
@@ -1255,78 +1022,6 @@ def _test_2d_mlp_with_nd_mesh(
             self.assertEqual(p.device_mesh.mesh_dim_names, ("dp", "tp"))
 
 
-class TestFullyShardHSDPTraining(FSDPTest):
```

Comment:
I would not consider HSDP as composability of parallelisms. FSDP2 itself needs to ensure HSDP is correct; it does not need to compose with some other API.